plugin/utils/casing_utils.py

```diff
@@ -32,6 +32,35 @@ def get_lower(s):
     return s[:1].lower() + s[1:]
 
 
+def to_snake(s):
```

Comment:
it would be nice to have a unit test for this function

Reply:
Totally fair. Will port the ones in [gapic-generator-python](https://github.com/googleapis/gapic-generator-python/blob/master/tests/unit/utils/test_case.py) (where I stole the original function from) over to this when I circle back to this work.

Reply:
Done.